pkg/config/vlan.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 NVIDIA CORPORATION
+// Copyright 2018-2019 Red Hat, Inc.
+// Copyright 2014 CNI authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/Mellanox/accelerated-bridge-cni/pkg/types"
+)
+
+func splitVlanIds(trunks []types.Trunk) ([]int, error) {
+	vlans := make(map[int]bool)
+	for _, item := range trunks {
+		var minID, maxID, id int
```

Comment:
are {item.MaxID, item.minID} and {item.ID} mutually exclusive ?

Reply:
It is possible to use all these options together
[{"id": 42, "minID": 100, "maxID": 103}] is fine, result will be 42,100,101,103

Reply:
The function is copied from ovs-cni https://github.com/k8snetworkplumbingwg/ovs-cni/blob/a140f3c96ab77a530ee8bc2a64f1a31a69ded145/pkg/plugin/plugin.go#L284. That's why it is in a separate file and has a stack of copyright notes.

Reply:
so there is ambiguity. its not good

`[{"id": 42, "minID": 100, "maxID": 103}] == [{"id": 42},  {"minID": 100, "maxID": 103}]`